rules/opentitan.bzl

```diff
@@ -4,14 +4,14 @@
 
 load("@rules_cc//cc:find_cc_toolchain.bzl", "find_cc_toolchain")
 load(
-    "@//rules:cc_side_outputs.bzl",
+    "//rules:cc_side_outputs.bzl",
```

Comment:
Does this work if you use `@lowrisc_opentitan`?

Reply:
I don't think it works. We setup opentitan as an local repository in the WORKSPACE file.
ex:
local_repository(
    name = "opentitan",
    path = "../opentitan", # local path to opentitan repo.
)
Before I did the patch, I tried to load a module from opentitan by `load("@opentitan//rules:linker.bzl", "ld_library")` and I get error: " Every .bzl file must have a corresponding package, but '//rules:cc_side_outputs.bzl' does not have one." 
After removing '@', the module could be loaded successfully.

Reply:
I see.  Can you try renaming your local_repository as `name = "lowrisc_opentitan"` (which is how our WORKSPACE names itself) and trying `@lowrisc_opentitan` on the labels in //rules/opentitan.bzl